### ISSUE TITLE
unused_types, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2385]
```abap
TYPES ty_bar TYPE i.
DATA bar TYPE REF TO data.
CREATE DATA bar TYPE ty_bar.
```
